.github/sync-repo-settings.yaml

```diff
@@ -15,6 +15,7 @@ branchProtectionRules:
       - units (8)
       - units (11)
       - cla/google
+      - bom-content-test
```

Comment:
@lqiu96 FYI, this file controls which jobs are required before merging to the protected main branch.